go-magic-link-example/static/success.html

```diff
@@ -15,7 +15,7 @@
             <div class="logged_in_div_left">
                 <div>
                     <h1>Your app,</h1>
-                    <h2>Enterprise Ready</h2>
```

Comment:
Could you also get rid of the text in the upper left and move the WorkOS logo to the left side here too?